vignettes/articles/wrangling_survey_responses.Rmd

```diff
@@ -0,0 +1,233 @@
+---
+title: "Wrangling form submissions"
+output: 
+  html_document:
+    toc: yes
+    toc_depth: 3
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  message = FALSE
+)
+```
+
+## Introduction
+
+This article demonstrates some techniques for cleaning and wrangling data returned by the `get_submissions` method of the `Kobo` and `Asset` classes. The vignette is broken down by the different question types. In general, users should make heavy use of the question options to give [custom column names](https://support.kobotoolbox.org/question_options.html#data-column-name) and labels for answer options. This will result in better behaving column names and make data cleaning easier.
+
+```{r setup}
+library(kbtbr)
+library(tidyr)
+library(stringr)
+library(dplyr)
+BASE_URL <- "https://kobo.correlaid.org"
+```
+
+```{r include=FALSE}
+kobo <-
+  Kobo$new(base_url_v2 = BASE_URL,
+           kobo_token = Sys.getenv("KBTBR_TOKEN"))
+vcr::use_cassette("kobo-asset-submissions-data", {
+  submissions_df <- kobo$get_submissions("aRo4wg5utWT7dwdnQQEAE7")
+})
+```
+
+
+```{r eval=FALSE}
+kobo <-
+  Kobo$new(base_url_v2 = BASE_URL,
+           kobo_token = Sys.getenv("KBTBR_TOKEN"))
+asset_id <- "aRo4wg5utWT7dwdnQQEAE7"
+submissions_df <- kobo$get_submissions(asset_id)
```

Comment:
these lines are not supposed to be run and are only displayed (hence `eval = FALSE` for the chunk option - if you knit instead of execute interactively it'll not run) . this is for the reader and not the knitter / developer so to say ;)